Sakari/feature/clans add fix update

Skoivumaki opened the pull request:
### Fixes #8
Work on this issue was started before new "branching strategy".
Added features/fixes
- Separate navbar for clan specific actions
- New page "Find Clans" for searching and showing all clans etc
- Clan create form updated
- Profile data is now updated to localstorage/redux store upon clan creation
- New clanApi query: getClanByIdWithPlayers

leolab1337 reviewed (changes requested):
Hello please fix some issues:

1)  We should not see that error , when we click the pagination's next button. It should be disabled or hidden ? 
![image](https://github.com/Alt-Org/Altzone-WebPages/assets/61798137/cc4ff946-06b3-485e-9dcb-407e1bd92fde)
2)  If user is not logged in, should he/she see that message ? 
![image](https://github.com/Alt-Org/Altzone-WebPages/assets/61798137/9b682703-f223-497f-9325-e2eb8163637e). May be U should use "canI" strategy 
3) same here , should the user see the menu if she/he is not logged in , 
![image](https://github.com/Alt-Org/Altzone-WebPages/assets/61798137/7479a428-1de4-404b-9d4a-7561f0702f55)
 
4) If the pc version has the own navbar , what about the mobile version ?

leolab1337 commented:
in my opinion you could try to modify the "IsActive" field. 
![image](https://github.com/Alt-Org/Altzone-WebPages/assets/61798137/ea39d155-2d13-4b7e-8d54-c14f23830a93)

1) Modify CSS to hide the inactive field. U could use classNames mods e.g. (see example shared/ui/button)
2) To handle the inactive field dynamically via the "canI" strategy

Skoivumaki commented:
New commit with requested fixes and other:
> 1. We should not see that error , when we click the pagination's next button. It should be disabled or hidden ?
>    ![image](https://private-user-images.githubusercontent.com/61798137/340118185-cc4ff946-06b3-485e-9dcb-407e1bd92fde.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg4OTYzMTQsIm5iZiI6MTcxODg5NjAxNCwicGF0aCI6Ii82MTc5ODEzNy8zNDAxMTgxODUtY2M0ZmY5NDYtMDZiMy00ODVlLTlkY2ItNDA3ZTFiZDkyZmRlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjIwVDE1MDY1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYwOGUyYjE1YjVmMGZjYmQ3YzMyN2NkN2IzN2Q5YmYzMjA4OGFmNTU5YmM0ZjgzNjYwZjc0NWQ5MDUwNjg3NzcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.9dvIpRP6USkWWdIMqsNYNVk9nFufzE7d1ADq_DNVZ1s)
- Fixed error with pagination buttons not properly disabling due to API response not providing paginationData if on last page.

However this will still cause errors if user uses search function and is on page 2. Search will fail even if it was viable. Needs to be fixed.
> 2. If user is not logged in, should he/she see that message ?

Was for debug now removed.
> 3. same here , should the user see the menu if she/he is not logged in ,
>    ![image](https://private-user-images.githubusercontent.com/61798137/340118612-7479a428-1de4-404b-9d4a-7561f0702f55.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg4OTYzMTQsIm5iZiI6MTcxODg5NjAxNCwicGF0aCI6Ii82MTc5ODEzNy8zNDAxMTg2MTItNzQ3OWE0MjgtMWRlNC00MDRiLTlkNGEtNzU2MWYwNzAyZjU1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjIwVDE1MDY1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdjMTUyZWMxMzNhMTVkODVlMGQxODNlYTY4OGNiYWM4OGExMjliN2Y3ZjFmN2Q4NDEzYjJiYjkzMDI3NzNiMmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.X0DOPqnq6_NFl9pm63NT-_kSGro4h8GrScWflQCWUp0)
> 4. If the pc version has the own navbar , what about the mobile version ?

 - Both mobile and PC now have their own clan navbars. 
 - "My Clan" button shouldn't be visible on navbar if player not in clan (canISeeOwnClan).
![image](https://github.com/Alt-Org/Altzone-WebPages/assets/123396118/7f440f6c-3578-4fa9-8d91-9645f4d4978b)
